zipkin-common/src/main/scala/com/twitter/zipkin/common/Dependencies.scala

```diff
@@ -16,10 +16,8 @@
  */
 package com.twitter.zipkin.common
 
-import com.twitter.algebird.{Monoid, Semigroup}
```

Comment:
The duties of this class are too simple to depend on algebird. Algebird shouldn't be a core dep, and now isn't.